packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -6,6 +6,78 @@ import 'package:flutter/widgets.dart';
 import 'bottom_tab_bar.dart';
 import 'theme.dart';
 
+/// Coordinates tab selection between a [CupertinoTabBar] and a [CupertinoTabScaffold].
+///
+/// The [index] property is the index of the selected tab. Changing its value updates
+/// the actively displayed tab of the [CupertinoTabScaffold] the [CupertinoTabController] controls,
+/// as well as the currently selected tab item of its [CupertinoTabBar].
+///
+/// {@tool sample}
+///
+/// [CupertinoTabController] can be used to switch tabs from ancestor widgets:
+///
+/// ```dart
+/// class MyCupertinoTabScaffoldPage extends StatefulWidget {
+///   @override
+///   _CupertinoTabScaffoldPageState createState() => _CupertinoTabScaffoldPageState();
+/// }
+///
+/// class _CupertinoTabScaffoldPageState extends State<MyCupertinoTabScaffoldPage> {
+///   final CupertinoTabController _controller = CupertinoTabController();
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return CupertinoTabScaffold(
+///       tabBar: CupertinoTabBar(
+///         items: <BottomNavigationBarItem> [
+///           // ...
+///         ],
+///       ),
+///       controller: _controller,
+///       tabBuilder: (BuildContext context, int index) {
+///         return Center(
+///           child: CupertinoButton(
+///             child: const Text('Go to first tab'),
+///             onPressed: () => _controller.index = 0,
+///           )
+///         );
+///       }
+///     );
+///   }
+/// }
+/// ```
+/// {@end-tool}
+///
+/// See also:
+/// * [CupertinoTabScaffold], a tabbed application root layout that can be controlled by a [CupertinoTabController].
```

Comment:
```dart
/// See also:
///
///  * [CupertinoTabScaffold], a tabbed application root layout that can be controlled by a [CupertinoTabController].
```

Otherwise the markdown might not render right